code/game/machinery/doors/brigdoors.dm

```diff
@@ -65,7 +65,7 @@
 	Radio.follow_target = src
 
 	pixel_x = ((dir & 3)? (0) : (dir == 4 ? 32 : -32))
-	pixel_y = ((dir & 3)? (dir ==1 ? 24 : -32) : (0))
+	pixel_y = ((dir & 3)? (dir ==1 ? 32 : -32) : (0))

```

Comment:
I couldn't see why this was done at `24`. It looks awful at `24`, and is inconsistent from all other directions.

This won't effect Box, as no north-facing ones are used.